statvar_imports/ireland_census/README.md

```diff
@@ -0,0 +1,53 @@
+# Ireland_Census
+
+- source urls: 
+`https://data.cso.ie/table/FY002`, `https://data.cso.ie/table/VSD30`, `https://data.cso.ie/table/VSD31`, `https://data.cso.ie/table/VSA30`, `https://data.cso.ie/table/VSA09`, `https://data.cso.ie/table/VSD24`, `https://data.cso.ie/table/VSA03`, `https://data.cso.ie/table/FY003B`, `https://data.cso.ie/table/FY001`, `https://data.cso.ie/table/FY031`, `https://data.cso.ie/table/CNA22`
+
+- how to download data: Download script (download_script.py).
+    To download the data, you'll need to use the provided download script, download_script.py. This script will automatically create an "input_files" folder where you should place the file to be processed. The script also requires a configuration file (config.py) to function correctly. We should run 'preprocess.py' script for removing duplicates data related suicides files. In suicides.csv input file contains state level data between the year 1950 and 2021 and the another input file 'suicides_with_aa1_aa2.csv' is having data from the year 2008 to latest which is creating duplicates. By executing the preprocess.py , we are removing the data from 2008 to 2021 and keeping the data from 2022 to the latest in the 'suicides_with_aa1_aa2.csv' and also keeping the suicides.csv file as it is after downloading for avoiding the conflicts.
```

Comment:
[Nit] I'd recommend surrounding any file name references with backticks (i.e; `) to keep them readable:

Example:

```
... use the provided download script, `download_script.py`. The script will automatically ...
```